src/index.html

```diff
@@ -5,9 +5,118 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.css">
+    <link rel="icon" href="./images/favicon.png">
+    <link href="https://fonts.googleapis.com/css?family=Ubuntu:300,400,400i,500,700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header">
+      <div class="header__top">
+        <div class="logo header__logo"></div>
+        <div class="header__phone">+1 234 555-55-55</div>
+        <div class="header__action"><a href="" class="header__link">call to order</a></div>
+      </div>
+
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item nav__item_active">Home</li>
+          <li class="nav__item"><a href="#" class="nav__link">About us</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Gallery</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Consultation</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Testimonials</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Contacts</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="section">
+        <div class="info">
+          <h1 class="info__heading">Miami <span class="bold-text">best</span> real estate <a href="#" class="link">here</a></h1>
```

Comment:
переноси для читаемости
```suggestion
          <h1 class="info__heading">
            <span>Miami</span>
            <span class="bold-text">best</span>
            <span>real estate</span>
            <a href="#" class="link">here</a>
          </h1>
```